src/app/modules/verification/verification.service.ts

```diff
@@ -363,6 +363,35 @@ export const sendNewOtp = ({
   })
 }
 
+export const checkShouldDisableVerifiedFields = (
```

Comment:
I think the function name is a bit misleading here, because it's not just a check right? it performs a whole bunch of actions including updating potentially multiple forms and contacting admins. maybe we can rename to `disableVerifiedFieldsIfRequired`? or something like that

Reply:
> there are 2 hard things in software engineering
your name is better; will change

Reply:
Resolved in 765e5ed